src/routes/settings/+page.svelte

```diff
@@ -140,27 +136,17 @@
 			<p class="p"><strong>About</strong></p>
 			<p class="p">
 				<strong>Hollama</strong> is a minimalistic web interface for
-				<Button
-					variant="link"
-					href="https://github.com/jmorganca/ollama/"
-					target="_blank"
-				>
+				<Button variant="link" href="https://github.com/jmorganca/ollama/" target="_blank">
 					Ollama
 				</Button>
 				servers. Code is available on
-				<Button
-					variant="link"
-					href="https://github.com/fmaclen/hollama"
-					target="_blank"
-				>
+				<Button variant="link" href="https://github.com/fmaclen/hollama" target="_blank">
 					Github
 				</Button>
 			</p>
 			<p class="p">
 				Made by
-				<Button variant="link" href="https://fernando.is" target="_blank">
-					@fmaclen
-				</Button>
+				<Button variant="link" href="https://fernando.is" target="_blank">@fmaclen</Button>
```

Comment:
I feel like most of these formatting changes are adding too much noise the PRs; and I probably it's mostly on me because I don't format on save 😬 

I'm thinking it might be good run a linter check in CI.

Reply:
https://github.com/fmaclen/hollama/issues/121

Reply:
I have a PR ready for this: https://github.com/fmaclen/hollama/pull/122

I'll wait to merge it after your other 2 PRs are merged first so I'll handle any potential conflicts.